app-dicts/myspell-it/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
```

Comment:
Eh, why? Since you didn't add yourself as maintainer, this is still a maintainer-needed package?

Reply:
Appoint myself as maintainer